src/tools/wasm-ctor-eval.cpp

```diff
@@ -165,41 +168,79 @@ class EvallingModuleInstance
   }
 };
 
+std::unique_ptr<Module> buildEnvModule(Module& wasm) {
```

Comment:
This could use some comment explaining what the creation of env does here, and why.

Reply:
Done.